### PR TITLE
Feature simulation configurator allows to add file parameters

### PR DESCRIPTION
**Pull request by ahmadswaid**
Thanks @schuelet for help in preferences tutorial:

`preferences: can be exported when starting KNIME as admin (sudo)
copy preference.epf to /opt/knime/knime_server/workflow_repository/config/client-profiles/executor
replace with executor.epf
add lines to knime.ini (/opt/knime/knime_server/knime_executor/knime.ini):
-profileLocation
http://127.0.0.1:8080/knime/rest/v4/profiles/contents
-profileList
executor`

Anyway, branch tested on my server, @schuelet could you please review this PR!

### CODE REVIEW

File: de.bund.bfr.knime.js/src/js/app/app.simulation.js [hunk 1]
```diff
@@ -660,7 +831,18 @@ class APPSimulation {
...
-						! _isNull( paramValue ) ? field.input.val( paramValue ) : null;
+					    if(field.param.dataType == 'FILE'){
+					    	
+					    	let paramId  = field.input.attr('id').split('_').pop();
+					    	let filesContainer = $('#filesArea-'+paramId);
+					    	let fileElement = $("<div ><hr/><p>"+ paramValue +"</div>");
+				    		
+                            
+		                    filesContainer.html(fileElement);
+						}
```
Comment (schuelet) [inline de.bund.bfr.knime.js/src/js/app/app.simulation.js:865]: @ahmadswaid this piece of code (line 834-842) does not work right now. The parameter values are not shown, regardless Windows or Linux. They are also removed from the simulation when clicking close and apply. Model taken for this is: https://knime.bfr.berlin/landingpage/DB/download/0f454533-c324-43a4-a1d1-5a997125a1ba?status=Any&repository=Any

image
image

**Review by schuelet — CHANGES_REQUESTED** [review]

**Comment by schuelet** [comment]
@ahmadswaid Ok, good news, the file parameter values are now visible on local execution. On server execution, the model executes and also takes new files via "add file" upload.
What is still an issue is the UI: On server execution, the button "add file" is still overlapping the file names and on clicking the details of a parameter, the information are pushed down, overlapping with the other parameters. 

image

**Comment by schuelet** [comment]
@ahmadswaid found 2 other issues:
1. When using the Simulation Configurator on Linux and saving a workflow, the portObject can't be opened after the Simulation Configurator (see image below). Error is: `ERROR FSK Simulation Configurator JS  3:5        Unable to clone input data at port 1 (FSK object): (was java.lang.NullPointerException) (through reference chain: de.bund.bfr.knime.fsklab.nodes.environment.ExistingEnvironmentManager["entries"])
`
2. Adding new files via "add file" button does seem to overwrite all other simulations with those filename values. I added a file called "conso_inputs4.csv" to a new simulation "MyNewSimulation" and yet all simulations have that file now. See json here: 
[simulations_after_adding_new_files.txt](https://github.com/SiLeBAT/FSK-Lab/files/10444263/simulations_after_adding_new_files.txt)

image

**Comment by schuelet** [comment]
@ahmadswaid: Tested the changes and most issues are fixed. Two issues remain:
1. if there are more then one FILE parameters, adding only one leaves the others empty:
image

2. the portObject still doesn't load when opening an exported workflow on another computer.

**Comment by ahmadswaid** [comment]
@schuelet issue No. 1 was solved
work is still in progress on issue No. 2

**Comment by schuelet** [comment]
@ahmadswaid The portObject Issue seems to be fixed. Unfortunately something broke with the upload, added (uploaded) files no longer appear in the workspace of the model. I checked the previous commit 42a1c05 and there the upload still worked.

**Review by schuelet — APPROVED** [review]

**Pull request merged**